geanyvc/src/geanyvc.c

```diff
@@ -1528,7 +1528,12 @@ vccommit_activated(G_GNUC_UNUSED GtkMenuItem * menuitem, G_GNUC_UNUSED gpointer
 	gint height;
 
 #ifdef USE_GTKSPELL
-	GtkSpell *speller = NULL;
+#if !GTK_CHECK_VERSION (3, 0, 0)
+//Since gtk3 will be default in future may be going this way would be better.
```

Comment:
comment is not very useful, but I agree adding compat layer for older version rather than newer